SPECS/kata-containers/versions-Update-Cloud-Hypervisor.patch

```diff
@@ -1,975 +0,0 @@
-From 9d286af7b4543b0af9e6db53c6c84e0731f61c09 Mon Sep 17 00:00:00 2001
```

Comment:
https://github.com/kata-containers/kata-containers/commit/9d286af7b4543b0af9e6db53c6c84e0731f61c09 merged in 3.1.0
<img width="475" alt="image" src="https://user-images.githubusercontent.com/30801614/234128640-a97c9957-ba94-4578-9820-d9aa2c43e433.png">